internal/renderer/renderer.go

```diff
@@ -487,6 +509,10 @@ func (r Renderer) RenderNode(w io.Writer, node ast.Node, entering bool) ast.Walk
 		r.table(w, node, entering)
 		noNewLine = false
 		fetchLinks = true
+	case *ast.HTMLBlock:
```

Comment:
Are `*ast.HTMLBlock`-s and `*ast.HTMLSpan`-s not children of a paragraph so we have to trigger their render manually?